express-api/src/services/buildings/buildingServices.ts

```diff
@@ -0,0 +1,121 @@
+import { Building } from '@/typeorm/Entities/Building';
+import { AppDataSource } from '@/appDataSource';
+import { ErrorWithCode } from '@/utilities/customErrors/ErrorWithCode';
+import { DeepPartial, FindOptionsOrder } from 'typeorm';
+import { BuildingFilter } from '@/services/buildings/buildingSchema';
+
+const buildingRepo = AppDataSource.getRepository(Building);
+
+/**
+ * @description          Adds a new building to the datasource.
+ * @param   building     Incoming building data to be added to the database.
+ * @returns {Response}   A 201 status and the building data added.
```

Comment:
This would return the building, not a response. The controller returns the response.

I'm seeing that this issue exists with the parcelServices.addParcel as well.